agent/smf/agent.xml

```diff
@@ -25,11 +30,10 @@
   </property_group>
 
   <property_group name='config' type='application'>
-    <propval name='datalink' type='astring' value='unknown' />
-    <propval name='gateway' type='astring' value='unknown' />
     <propval name='dataset' type='astring' value='' />
     <propval name='listen_addr' type='astring' value='127.0.0.1' />
     <propval name='listen_port' type='astring' value='17000' />
+    <!-- TODO: Are 'uuid' and 'nexus' being used? -->
     <propval name='uuid' type='astring' value='' />
     <propval name='nexus' type='astring' value='127.0.0.1:12221' />
```

Comment:
They might not be used -- I think `nexus` might have been a relic from before using DNS to find the Nexus address